dashboards/flask_webapp.json

```diff
@@ -0,0 +1,976 @@
+{
+  "__inputs": [
+    {
+      "name": "DS_PROMETHEUS",
+      "label": "Prometheus",
+      "description": "",
+      "type": "datasource",
+      "pluginId": "prometheus",
+      "pluginName": "Prometheus"
+    }
+  ],
+  "__requires": [
+    {
+      "type": "grafana",
+      "id": "grafana",
+      "name": "Grafana",
+      "version": "6.0.2"
```

Comment:
In https://github.com/rycus86/prometheus_flask_exporter/blob/master/examples/sample-signals/docker-compose.yml#L28 the Grafana version was `5.1.0`, any reason you picked version 6 instead?

Reply:
No. I just copied this part from https://github.com/gangsta/jenkins-prometheus-grafana/blob/master/provisioning/dashboards/Jenkins_%20Performance.json since grafana central request some artifacts